cmd/transportRequestUploadCTS.go

```diff
@@ -71,6 +73,25 @@ func runTransportRequestUploadCTS(
 
 	log.Entry().Debugf("Entering 'runTransportRequestUpload' with config: %v", config)
 
+	transportRequestID := config.TransportRequestID
+
+	if len(transportRequestID) == 0 {
+
+		from := "origin/master"                         //TODO: make configurable
+		to := "HEAD"                                    //TODO: make configurable
+		transportRequestIdLabel := "TransportRequestID" //TODO: make configurable
+
+		log.Entry().Infof("%s not provided by configuration. Traversing commit history, range: '%s..%s'", transportRequestIdLabel, from, to)
+		var err error
+		transportRequestID, err = transportrequest.FindIDInRange("TransportRequest", from, to)
```

Comment:
Isn't the label "TransportRequest" configurable?

Reply:
Needs to be configurable, yes. PR is in status draft since it is not ready to merge for exactly reasons like that ... But thanks for pointing to it ...